includes/pages/user_shifts.php

```diff
@@ -117,23 +119,49 @@ function user_shifts() {
         $msg .= error(_('Please select a shifttype.'), true);
       }
 
-      if (isset($_REQUEST['start']) && $tmp = DateTime::createFromFormat("Y-m-d H:i", trim($_REQUEST['start'])))
+      if (isset($_REQUEST['start']) && $tmp = DateTime::createFromFormat("Y-m-d", trim($_REQUEST['start'])))
         $start = $tmp->getTimestamp();
       else {
         $ok = false;
-        $msg .= error(_("Please enter a valid starting time for the shifts."), true);
+        error(_('Please select a start date.'));
       }
-
-      if (isset($_REQUEST['end']) && $tmp = DateTime::createFromFormat("Y-m-d H:i", trim($_REQUEST['end'])))
+    
```

Comment:
I don't complaint about new line. I complaint about "trailing space".
The blank line for separate block of code should contain only "\n", not "\n space space space".